src/lib/shim/shim_api_c.h

```diff
@@ -10,15 +10,20 @@
 #include <netdb.h>
 #include <stdarg.h>
 
-/// This module defines functions that can be called by external (preloaded)
-/// libraries that are linked to the shim. Those libraries should only call
-/// functions defined here after including this header file.
+#include "lib/shim/shim_api.h"
+
+/// This module defines *C implementations* of functions that can be called by
+/// external (preloaded) libraries that are linked to the shim. Those libraries
+/// should only call functions defined here after including this header file.
+///
+/// The public interfaces to access these are defined in Rust, without prefix
+/// `shim_api_` instead of `shimc_api`.
```

Comment:
Should this be "with" rather than "without"?

Reply:
Oops; yes :)